test_util/terraform_test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+if [ $# -lt 1 ]; then
+    echo "Usage: $0 <test group>"
+    exit 1
+fi
+
+set +e
+set -x
+set -o pipefail
+
+MASTER_IP=$(./terraform output --json masters-ips |jq -r '.value[0]')
+ssh_user=$(./terraform output --json -module dcos.dcos-infrastructure masters.os_user |jq -r '.value')
+ssh_user=${ssh_user:-$(./terraform output --json -module dcos.dcos-infrastructure masters.admin_username |jq -r '.value')}
+
+SSH="ssh  -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ./id_rsa ${ssh_user}@$MASTER_IP"
+PYTEST='source /opt/mesosphere/environment.export && cd `find /opt/mesosphere/active/ -name dcos-integration-test* | sort | tail -n 1` && '
```

Comment:
So, this invokes the _test runner process_ on a master node (_the_ master node in a single master node cluster), right?

Maybe add a comment to the top of this file which briefly lays that out.

Reply:
Can you rename `PYTEST` to `PYTEST_EXECUTABLE_ABS_PATH` if that is what it is?